apps/blehr/src/main.c

```diff
@@ -162,11 +165,17 @@ blehr_gap_event(struct ble_gap_event *event, void *arg)
         if (event->connect.status != 0) {
             /* Connection failed; resume advertising */
             blehr_advertise();
+            conn_handle = 0;
+        }
+        else {
+          conn_handle = event->connect.conn_handle;
         }
+
         break;
 
     case BLE_GAP_EVENT_DISCONNECT:
         MODLOG_DFLT(INFO, "disconnect; reason=%d\n", event->disconnect.reason);
+        conn_handle = 0; // reset conn_handle
```

Comment:
0 is a valid connection handle, we have symbol defined iirc in GAP for invalid conn handle
also do not use C++ style comments

Reply:
Could you point me to the symbol definition `iirc` ? I could not find it gap services.